releases/keycloak-gatekeeper.yaml

```diff
@@ -0,0 +1,113 @@
+#      host: dashboard.us-west-2.corp.cpco.io # The DNS name of the public endpoint
+#      useTLS: false  # True to generate a TLS certificate and use an HTTPS endpoint
+#      upstream: https://kubernetes-dashboard.kube-system.svc.cluster.local  # The service URL
+#      skipUpstreamTlsVerify: true # True to skip the validity check of the TLS certificate of the upstream URL
+#      rules: # Array of access control rules
+#        - "uri=/*|roles=kube-admin"
+#      replicas: 1 # number of replicas
+#    - name: forecastle
+#      debug: true
+#      host: forecastle.us-west-2.corp.cpco.io
+#      upstream: http://forecastle.kube-system.svc.cluster.local
+#      rules:
+#        - "uri=/*|roles=kube-admin,user|require-any-role=true"
+######
+
+#
+environments:
+  default:
+    values:
+     - ../gatekeeper.yaml
+
+releases:
+
+#######################################################################################
+## keycloak-gatekeeper (formerly keycloak-proxy)                                     ##
+## keycloak-gatekeeper is an open source identity-aware proxy                        ##
+#######################################################################################
+
+#
+# References:
+#   - https://github.com/gabibbo97/charts/tree/master/charts/keycloak-gatekeeper
+#   - https://hub.helm.sh/charts/gabibbo97/keycloak-gatekeeper
+#   - https://www.keycloak.org/
+#
+{{ range $index, $service := .Environment.Values.services }}
+- name: "key-gate-{{- $service.name }}"
+  namespace: "kube-system"
+  labels:
+    chart: "keycloak-gatekeeper"
+    repo: "gabibbo97"
+    component: "iap"
+    namespace: "kube-system"
+    vendor: "keycloak"
+    default: "false"
+  chart: "gabibbo97/keycloak-gatekeeper"
+  version: "1.1.1"
+  wait: false
+  installed: true
+  values:
+  - nameOverride: "key-gate-{{- $service.name }}"
+    fullNameOverride: "key-gate-{{- $service.name }}"
+    image:
+      tag: 5.0.0
+      pullPolicy: "IfNotPresent"
+    debug: {{ index $service "debug" | default "false" }}
+    replicas: {{ index $service "replicas" | default 1 }}
+    ingress:
+      enabled: '{{ env "KEYCLOAK_GATEKEEPER_INGRESS_ENABLED" | default "true" }}'
+      annotations:
+        kubernetes.io/ingress.class: nginx
+        nginx.ingress.kubernetes.io/proxy-buffer-size: "16k"
+        kubernetes.io/tls-acme: "{{ index $service "useTLS" | default "false" }}"
+        external-dns.alpha.kubernetes.io/target: '{{ requiredEnv "NGINX_INGRESS_HOSTNAME" }}'
+        external-dns.alpha.kubernetes.io/ttl: "60"
+        {{- if index $service "portalName"  }}
+        forecastle.stakater.com/expose: "true"
+        forecastle.stakater.com/appName: "{{ $service.portalName }}"
+        {{- end }}
+      hosts:
+      - {{ $service.host }}
+      # tls:
+      #  - secretName: keycloak-http-tls
+      #    hosts:  [{{ env "KEYCLOAK_INGRESS_HOSTS" }}]
+    discoveryURL: '{{- requiredEnv "KOPS_OIDC_ISSUER_URL" }}'
```

Comment:
This only applies to the kubernetes dashboard, no?

Reply:
No, every instance of Gatekeeper needs to communicate with Keycloak